packages/react/src/components/Switch/Switch.js

```diff
@@ -24,6 +25,12 @@ const Switch = props => {
     ...other
   } = props;
 
+  const tabRef = useRef();
+
+  useEffect(() => {
+    selected && tabRef.current.focus();
+  });
```

Comment:
Would this need to be synchronized with `selected`? e.g.

```
 useEffect(() => {
    selected && tabRef.current.focus();
 }, [selected]);
```

Also, do we need to "clean-up" our focus with a return function? (Definitely not sure, just thinking out loud)

Reply:
We talked about this offline, but we're not gonna sweat this right now, because it's probably not a problem. 🏄

Reply:
Maybe:

```js
const ref = useRef(null);

useEffect(() => {
  if (isSelected && ref.current) {
    ref.current.focus();
  }
}, [isSelected, ref.current]);
```

Seems like we need to cover:

| `selected` | `ref.current` | Result |
| - | - | - |
| false | null | no focus |
| true | null | no focus |
| false | Node | no focus |
| true | Node | focus |